src/app/core/services/phase.service.ts

```diff
@@ -41,7 +41,11 @@ export class PhaseService {
     phaseModeration: ''
   };
 
-  constructor(private githubService: GithubService, private labelService: LabelService, private repoCreatorService: RepoCreatorService) {}
+  private phasesRequiringClosedIssues: Set<Phase>;
+
+  constructor(private githubService: GithubService, private labelService: LabelService, private repoCreatorService: RepoCreatorService) {
+    this.phasesRequiringClosedIssues = new Set<Phase>([Phase.phaseBugReporting, Phase.phaseBugTrimming]);
```

Comment:
I'll approve this change.

However, do consider defining it as a constant outside of the class to maintain consistency with the rest of the codebase.  

Another consideration is whether to use array instead of `Set` due to the `Set` performance and memory overhead with small number of elements. You can check out [this discussion](https://stackoverflow.com/a/75514478) for more details."